lib/launcher.ts

```diff
@@ -243,7 +241,13 @@ let initFn = function(configFile: string, additionalConfig: Config) {
           }
         }
 
-        let deferred = q.defer<any>();  // Resolved when all tasks are completed
+        let deferredResolve: () => void;
```

Comment:
Do we ever return this deferred? If not, we should just wrap the following code in a promise